src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -0,0 +1,1240 @@
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['installationParam']
+    del device_b['measCapability']
+
+    conditionals = [conditionals_b]
+
+    # Create the actual config.
+    config = {
+        'registrationRequests': [device_a, device_b],
+        'expectedResponseCodes': [(103,), (0,)],
+        'conditionalRegistrationData': conditionals
+        # TODO
+        # Need to add data base configurations
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_FDB_7_default_config)
+  def test_WINNF_FT_S_FDB_7(self, config_filename):
+    """FCC ID Database Update."""
+
+    # Load the configuration file
+    config = loadConfig(config_filename)
+
+    registration_requests = {
+        'registrationRequest': config['registrationRequests']
+    }
+
+    # Step 1: Pre-load conditional registration data for CBSD.
+    if ('conditionalRegistrationData' in config)\
+      and (config['conditionalRegistrationData']):
+      self._sas_admin.PreloadRegistrationData(
+          config['conditionalRegistrationData'])
+
+    # Send registration request for CBSD 'C' with FCC ID 'F_ID' to SAS UUT.
+    registration_responses = self._sas.Registration(registration_requests)['registrationResponse']
+
+    # Check registration response,
+    # responseCode should be 103(INVALID_VALUE)
+    for resp in registration_responses:
+      self.assertEqual(resp['response']['responseCode'], 103)
+
+    del registration_responses
+
+    # TODO
+    # Step 2: Create FCC ID database which includes at least one FCC ID 'F_ID'.
+
+    # Step 3: Trigger daily activities
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+
+    # Step 4: Send again the same registration request for CBSD 'C'
+    # with FCC ID 'F_ID' to SAS UUT.
+    registration_responses = self._sas.Registration(registration_requests)['registrationResponse']
+
+    # Check registration response,
+    # responseCode should be 0(SUCCESS)
+    for resp in registration_responses:
+      self.assertEqual(resp['response']['responseCode'], 0)
+
+    del registration_responses
+
+    # TODO
+    # Step 5: Modify FCC ID database record with FCC ID 'F_ID'.
+
+    # Step 6: Trigger daily activities
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+
+    # Step 7: Send again the same registration request for CBSD 'C'
+    # with FCC ID 'F_ID' to SAS UUT.
+    registration_responses = self._sas.Registration(registration_requests)['registrationResponse']
+
+    # Check the registration response code is as expected (SUCCESS or INVALID_VALUE).
+    for resp in registration_responses:
```

Comment:
Instead of `[1]`, we should use the index corresponding to this reg request.

Reply:
FDB.7     Using the index corresponding to each registration request against the   registration response.